src/App.js

```diff
@@ -10,6 +10,7 @@ import Wallet from './components/Wallet'
 import useAppLogic from './app-logic'
 import { useWallet } from './providers/Wallet'
 import useSelectedProposal from './hooks/useSelectedProposal'
+import NetworkErrorModal from './components/NetworkErrorModal'
```

Comment:
Could we move this to above where we have the component imports? 

We generally order the imports in the following way:

- External library imports
- Component imports 
- Hook imports
- local utilities, library imports

And we also order them in alphabetical order 😄

Reply:
Hey Fabri, 

Thank you for the explanation!!, I'll change these things.
I've just added prettier to my vsCode, will be checking with Rodri in case some configuration missing.